src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -20,30 +20,36 @@ public class Climber extends SubsystemBase {
     private double m_rightPosition;
     private double m_leftVelocity;
     private double m_rightVelocity;
-    private double m_leftEngagedPosition;
-    private double m_rightEngagedPosition;
+    // private double m_leftEngagedPosition;
+    // private double m_rightEngagedPosition;
     private boolean m_leftHookReadyToEngage = false;
     private boolean m_rightHookReadyToEngage = false;
     private boolean m_leftHookEngaged = false;
     private boolean m_rightHookEngaged = false;
-    // private boolean m_leftHookComplete = false;
-    // private boolean m_rightHookComplete = false;
+    private boolean m_leftHookComplete = false;
+    private boolean m_rightHookComplete = false;
     private double m_rollAngle;
 
     // Constants to be used in this class
-    private static final double WINCH_GEAR_RATIO = 1.0/15.0;
-    private static final double NOMINAL_WINCH_DIAMETER = Units.inchesToMeters(0.75);
-    private static final double NOMINAL_INCHES_PER_ROTATION = Math.PI * NOMINAL_WINCH_DIAMETER * WINCH_GEAR_RATIO;
+    // private static final double WINCH_GEAR_RATIO = 1.0/15.0;
```

Comment:
Helpful to save the commented lines, or can we delete?

Reply:
Think we should save for now.